Core.DomainModel/ItContract/ItContract.cs

```diff
@@ -543,6 +543,8 @@ public void ClearMasterReference()
 
         #endregion
 
+        public virtual ICollection<DataProcessingRegistration> AssociatedDataProcessingRegistrations { get; set; }
```

Comment:
better naming is preferred because it is not just "associated"... it's `DataProcessingRegistrationsWhereContractIsMainContract`